Experimental/DedicatedGameServer/ProjectSettings/InputManager.asset

```diff
@@ -1,946 +1,5 @@
 %YAML 1.1
```

Comment:
Are there compatibility reasons for why this project wouldn't target the new Input system only?

Reply:
There is a bug in the Input System package for which [the developers recommend ](https://unity.slack.com/archives/C06GKHGMUN8/p1712170177241359?thread_ts=1712053238.013589&cid=C06GKHGMUN8)to keep the "Both" option for now in case users add UGUI to a project.